src/_pytest/mark/evaluate.py

```diff
@@ -10,25 +10,14 @@
 from ..outcomes import fail
 from ..outcomes import TEST_OUTCOME
 from .structures import Mark
-from _pytest.config import Config
 from _pytest.nodes import Item
-from _pytest.store import StoreKey
 
 
-evalcache_key = StoreKey[Dict[str, Any]]()
+def compiled_eval(expr: str, d: Dict[str, object]) -> Any:
```

Comment:
My preference is to inline this code directly into `_istrue()`, but it's OK if you prefer this way.